lib/wallet/test/unit/Cardano/Wallet/Shelley/TransactionSpec.hs

```diff
@@ -991,8 +991,18 @@ decodeSealedTxSpec = describe "SealedTx serialisation/deserialisation" $ do
         , "b77b47f2ddb31c19326b87ed6f71fb9a27133ad51b000000e8d4a510000e80a0f5f6"
         ]
 
-feeEstimationRegressionSpec :: Spec
-feeEstimationRegressionSpec = describe "Regression tests" $ do
+feeEstimationRegressionSpec :: AnyRecentEra -> Spec
+feeEstimationRegressionSpec (AnyRecentEra era) =
+    case era of
+        RecentEraConway ->
+            feeEstimationRegressionSpecInner @Cardano.ConwayEra era
+        RecentEraBabbage ->
+            feeEstimationRegressionSpecInner @Cardano.BabbageEra era
+
+feeEstimationRegressionSpecInner
```

Comment:
It seems unnecessary to define `feeEstimationRegressionSpecInner` separately from `feeEstimationRegressionSpec` to me. I'll create another PR

Reply:
https://github.com/cardano-foundation/cardano-wallet/pull/4207

Reply:
@Anviking Thanks for the suggestion! I've cherry-picked your suggested commit into this branch. 👍🏻